### PR TITLE
[#533] code review

### LINKED ISSUE #533: Code review

### PR DESCRIPTION
**Pull request by MattD8957**
Closes #533

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -25,5 +25,6 @@
       }
     },
   ],
-  "java.test.defaultConfig": "WPIlibUnitTests"
+  "java.test.defaultConfig": "WPIlibUnitTests",
+  "java.format.settings.url": ".vscode/eclipse-java-google-style.xml"
```
Comment (WindowsVistaisCool) [inline .vscode/settings.json:29]: this should be set to `.github/sun_checks.xml`

File: src/main/java/frc/robot/Constants.java [hunk 2]
```diff
@@ -769,28 +682,7 @@ public class LEDsConstants {
...
-            BAD_POSE(13),
-            HAS_PIECE(14),
-            HAS_VISION(15),
-            DEFAULT(16),
-            PIVOT_BOTTOM_SWITCH(17),
-            PIVOT_TOP_SWITCH(18),
-            COLLECTOR_BEAMBREAK(19),
-            INDEXER_ENTER_BEAMBREAK(20),
-            INDEXER_EXIT_BEAMBREAK(21);
+            CUSTOMCONTROL(0), DISABLED(1), EMERGENCY(2), START(3), GOOD_POSE(4), COLLECT_PLANNED(5), COLLECTED(6), SHOT(7), FINISHED_CLIMB(8), SHOOTING(9), COLLECTING(10), CHASING(11), CLIMBING(12), BAD_POSE(13), HAS_PIECE(14), HAS_VISION(15), DEFAULT(16), PIVOT_BOTTOM_SWITCH(17), PIVOT_TOP_SWITCH(18), COLLECTOR_BEAMBREAK(19), INDEXER_ENTER_BEAMBREAK(20), INDEXER_EXIT_BEAMBREAK(21);
```
Comment (WindowsVistaisCool) [inline src/main/java/frc/robot/Constants.java:685]: not again 😭

**Review by WindowsVistaisCool — APPROVED** [review]
if thunder changes are okay, fix your linter, then im happy

**Comment by MattD8957** [comment]
I'm still cleaning, also the LED thing got fixed. It needs to be that or you need to change the one online because those settings are the ones that make the formater funny.

File: src/main/java/frc/robot/Constants.java (hunk 2 above)
Comment (MattD8957) [reply src/main/java/frc/robot/Constants.java:685]: already fixed

File: .vscode/settings.json (hunk 1 above)
Comment (MattD8957) [reply .vscode/settings.json:29]: @WindowsVistaisCool You need to change the online version then, because this is the fix for redhat formatter

**Pull request merged**